lib/src/footer/Footer.tsx

```diff
@@ -150,11 +143,10 @@ const BottomLinks = styled.div`
   & > span:last-child span {
     display: none;
   }
-  margin: ${(props) => (props.refSize <= responsiveSizes.mobileLarge ? "40px 0 40px 0" : "")};
+  align-self: center;
 `;
 
 const ChildComponents = styled.div`
-  min-height: 15px;
```

Comment:
`min-height` is 16px according to the guidelines.

Reply:
![47e160bdd040410030944251f12c5d53](https://user-images.githubusercontent.com/44420072/159249325-c1ff046f-a920-419e-aae6-e9503496766b.png)

Reply:
This min-height is not doing anything. However, it is applying a margin-top of 16px (line 134).

Reply:
Done